{
    "id": 210715,
    "name": "CVE-2023-28858",
    "cwe": "CWE-193",
    "modified": "2023-05-17T15:07:38",
    "published": "2023-03-26T17:15:06",
    "status": "Analyzed",
    "summary": "redis-py before 4.5.3 leaves a connection open after canceling an async Redis command at an inopportune time, and can send response data to the client of an unrelated request in an off-by-one manner. NOTE: this CVE Record was initially created in response to reports about ChatGPT, and 4.3.6, 4.4.3, and 4.5.3 were released (changing the behavior for pipeline operations); however, please see CVE-2023-28859 about addressing data leakage across AsyncIO connections in general.",
    "cvss3": {
        "cvss": 3.7,
        "exploitability_score": 2.2,
        "impact_score": 1.4,
        "vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:L/I:N/A:N"
    },
    "cvss2": null,
    "cpes": [
        {
            "id": 1052663
        },
        {
            "id": 1052664
        },
        {
            "id": 1052665
        },
        {
            "id": 1052666
        },
        {
            "id": 1052667
        },
        {
            "id": 1052668
        },
        {
            "id": 1052669
        },
        {
            "id": 1052672
        },
        {
            "id": 1052673
        },
        {
            "id": 1052674
        },
        {
            "id": 1052676
        },
        {
            "id": 1052677
        },
        {
            "id": 1052678
        },
        {
            "id": 1052679
        },
        {
            "id": 1052680
        },
        {
            "id": 1052681
        },
        {
            "id": 1052682
        },
        {
            "id": 1052683
        },
        {
            "id": 1052684
        },
        {
            "id": 1052685
        },
        {
            "id": 1052686
        },
        {
            "id": 1052687
        }
    ]
}